src/utils/exr-parser.worker.js

```diff
@@ -17,102 +17,104 @@
+        ctx.postMessage({
+            jobId: jobId,
+            success: true,
+            image: image
+        }, [image.data.buffer]);
+    }
+    catch (error) {
         console.log('Error: ', error);
-        self.postMessage({
-            jobId,
+        // eslint-disable-next-line no-restricted-globals
+        ctx.postMessage({
+            jobId: jobId,
             success: false,
             message: error.toString()
         });
     }
 }
-
+// tslint:disable-line:no-any
 function parseExr(data) {
     console.time('Decoding EXR'); // tslint:disable-line
-    let exrImage = null; // tslint:disable-line:no-any
+    var exrImage = null; // tslint:disable-line:no-any
     try {
-        exrImage = OpenEXR.loadEXRStr(data);
-        const channels = exrImage.channels();
-        const {
-            width,
-            height
-        } = exrImage;
-        let nChannels = channels.length;
-        let exrData;
+        exrImage = openEXR.loadEXRStr(data);
+        var channels = exrImage.channels();
+        var width = exrImage.width, height = exrImage.height;
+        var nChannels = channels.length;
+        var exrData = void 0;
         if (nChannels === 1) {
-            const z = exrImage.plane(exrImage.channels()[0]);
+            var z = exrImage.plane(exrImage.channels()[0]);
             exrData = new Float32Array(width * height);
-            for (let i = 0; i < width * height; i++) {
+            for (var i = 0; i < width * height; i++) {
                 exrData[i] = z[i];
             }
-        } else if (exrImage.channels().includes('R') &&
+        }
+        else if (exrImage.channels().includes('R') &&
             exrImage.channels().includes('G') &&
             exrImage.channels().includes('B')) {
-            const r = exrImage.plane('R');
-            const g = exrImage.plane('G');
-            const b = exrImage.plane('B');
+            var r = exrImage.plane('R');
+            var g = exrImage.plane('G');
+            var b = exrImage.plane('B');
             exrData = new Float32Array(width * height * 3);
-            for (let i = 0; i < width * height; i++) {
+            for (var i = 0; i < width * height; i++) {
                 exrData[i * 3] = r[i];
                 exrData[i * 3 + 1] = g[i];
                 exrData[i * 3 + 2] = b[i];
             }
             nChannels = 3;
-        } else {
+        }
+        else {
             throw new Error('EXR image not supported');
         }
         return {
-            height,
-            width,
-            nChannels,
+            height: height,
+            width: width,
+            nChannels: nChannels,
             data: exrData,
-            type: 'HdrImage',
+            type: 'HdrImage'
         };
-    } finally {
+    }
+    finally {
         if (exrImage) {
-            exrImage.delete();
+            exrImage["delete"]();
         }
         console.timeEnd('Decoding EXR'); // tslint:disable-line
     }
-};
+}
+;
+
```

Comment:
Note I had rewritten this file as a typescript file, then copied the js output back to this file, so there are some things changed here we may want to change back, like const -> var, it should be const.  Just noting that.